sdks/python/apache_beam/utils/counters.py

```diff
@@ -132,7 +132,7 @@ class Counter(object):
   MEAN = cy_combiners.MeanInt64Fn()
 
   # Distribution Accumulator Fn
-  DISTRIBUTION = cy_combiners.DistributionCounterFn()
+  DISTRIBUTION = cy_combiners.DataflowDistributionCounterFn()
```

Comment:
<!--new_thread; commit:dd3ae5364c173a99c8a15f7e80960b9307993f7c; resolved:0-->
Please rename `DISTRIBUTION` to something more that doesn't imply this is used for all distribution metrics.

Reply:
Sure.  It's also better if later we can generalize this for other cases (e.g. when Beam on Flink / portability emerges), so could you also file a JIRA for generalizing usage of this distribution counter, and reference it here in a comment?